packages/hashmyfiles.vm/hashmyfiles.vm.nuspec

```
@@ -2,12 +2,13 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
   <metadata>
     <id>hashmyfiles.vm</id>
-    <version>0.0.0.20250219</version>
-    <description>HashMyFiles is small utility that allows you to calculate the MD5 and SHA1 hashes of one or more files in your system. You can easily copy the MD5/SHA1 hashes list into the clipboard, or save them into text/html/xml file.</description>
+    <version>0.0.0.20250505</version>
+    <description>HashMyFiles is small utility that calculates and easily copies or saves MD5/SHA1 hashes for multiple files.</description>
```

HashMyFiles support more hash types and in the package we have changed the default columns to only include MD5 and SHA256. I propose the following enhancement:
```suggestion
    <description>HashMyFiles calculates and exports various file hashes (MD5, SHA256, etc.) to clipboard and multiple file formats.</description>
```